_monitoring-plugins/ad/api.md

```diff
@@ -668,1043 +1929,1016 @@ POST _plugins/_anomaly_detection/detectors/results/_search
-            },
-            "detection_interval": {
-              "period": {
-                "unit": "Minutes",
-                "interval": 10
-              }
-            },
-            "name": "test-historical-detector",
-            "filter_query": {
-              "match_all": {
-                "boost": 1
-              }
-            },
-            "shingle_size": 8,
-            "user": {
-              "backend_roles": [
-                "admin"
-              ],
-              "custom_attribute_names": [],
-              "roles": [
-                "all_access",
-                "own_index"
-              ],
-              "name": "admin",
-              "user_requested_tenant": "__user__"
-            },
-            "detector_type": "HISTORICAL_SINGLE_ENTITY"
-          },
+          "detector_id": "Zi5zTXwBwf_U8gjUTfJG",
+          "confidence": 0.918886275269358,
+          "model_id": "Zi5zTXwBwf_U8gjUTfJG_entity_error16",
+          "schema_version": 4,
+          "anomaly_score": 1.1093755891885446,
+          "execution_start_time": 1633388475001,
+          "data_end_time": 1633388414989,
+          "data_start_time": 1633388114989,
+          "feature_data": [
+            {
+              "feature_id": "ZS5zTXwBwf_U8gjUTfIn",
+              "feature_name": "test_feature",
+              "data": 0.532
+            }
+          ],
+          "execution_end_time": 1633388475014,
           "user": {
             "backend_roles": [
               "admin"
             ],
             "custom_attribute_names": [],
             "roles": [
-              "all_access",
-              "own_index"
+              "own_index",
+              "all_access"
             ],
             "name": "admin",
             "user_requested_tenant": "__user__"
-          }
+          },
+          "anomaly_grade": 0.031023547546561225,
+          "entity": [
+            {
+              "name": "error_type",
+              "value": "error16"
+            }
+          ]
         }
-      }
+      },
+      ...
     ]
   }
 }
 ```
 
+You can run historical analysis as many times as you like. So, multiple tasks might exist for the same detector.
 
----
-
-## Delete detector
-Introduced 1.0
-{: .label .label-purple }
-
-Deletes a detector based on the `detector_id`.
-To delete a detector, you need to first stop the detector.
-
-#### Request
-
-```json
-DELETE _plugins/_anomaly_detection/detectors/<detectorId>
-```
-
-
-#### Sample response
-
-```json
-{
-  "_index" : ".opendistro-anomaly-detectors",
-  "_type" : "_doc",
-  "_id" : "m4ccEnIBTXsGi3mvMt9p",
-  "_version" : 2,
-  "result" : "deleted",
-  "forced_refresh" : true,
-  "_shards" : {
-    "total" : 2,
-    "successful" : 2,
-    "failed" : 0
-  },
-  "_seq_no" : 6,
-  "_primary_term" : 1
-}
-```
-
-
----
-
-## Update detector
-Introduced 1.0
-{: .label .label-purple }
+You can search for the latest historical batch task first and then search the historical batch task results.
```

Comment:
Should we add how to search for the latest historical batch task first? If yes, please sync up with Yaliang for queries.